contracts/contracts/vault/VaultAdmin.sol

```diff
@@ -133,6 +133,19 @@ contract VaultAdmin is VaultStorage {
         assetDefaultStrategies[_asset] = _strategy;
     }
 
+    /**
+     * @dev Set maximum amount of OUSD that can at any point be minted and deployed
+     * to strategy (used only by ConvexOUSDMetaStrategy for now).
+     * @param _threshold OUSD amount with 18 fixed decimals.
+     */
+    function setNetOusdMintForStrategyThreshold(uint256 _threshold)
+        external
+        onlyGovernor
+    {
+        netOusdMintForStrategyThreshold = _threshold;
+        emit NetOusdMintForStrategyThresholdChanged(_threshold);
```

Comment:
In use, the OUSD AMO is likely to mostly operate in one direction for either long periods of time, either mostly minting OUSD, or mostly redeeming OUSD. Because of this, governance will need keep extending the netOusdMintForStrategyThreshold. However, this high netOusdMintForStrategyThreshold will mean that that while it might only allow a small change in one direction in the typical direction, it could allow a truly massive change back the other way. For example, if it had mostly been used for redeems, and had done a net 105 million burns so far, we might want to set the limit to 115 million to enable 10 million more burns. However, this would mean that it could also be permitted to mint 220 million OUSD. I think the solution to this is that whenever governance changes the limit, we should reset the `netOusdMintForStrategy` back to zero. This will mean that the new limit based on where things currently are.

Reply:
Thanks @DanielVF! And sorry for overlooking this in your CR sum up comment. You have already laid all this out there. Fix for it here: https://github.com/OriginProtocol/origin-dollar/pull/1002/commits/302ea687c19c910dff8202086a7b8d7f2ddc2de2

Also another thing we gain with this is that we can actually reduce the threshold amount and not worry that the current netOusdMinted counter is too much near one or the other limit. Which would make it hard to make the threshold smaller.